onnx/test/parser_test.py

```diff
@@ -72,6 +72,49 @@ def test_parse_model_error(self) -> None:
             onnx.parser.ParseError, lambda: onnx.parser.parse_model(input)
         )
 
+    def test_parse_function_with_attributes(self) -> None:
+        input = """
+            <
+            ir_version: 9,
+            opset_import: [ "" : 15, "custom_domain" : 1],
+            producer_name: "FunctionProtoTest",
+            producer_version: "1.0",
+            model_version: 1,
+            doc_string: "A test model for model local functions."
+          >
+         agraph (float[N] x) => (float[N] out)
+         {
+            out = custom_domain.Selu<gamma=2.0, gamma=3.0>(x)
```

Comment:
gamma or alpha? Maybe this should raise an error.

Reply:
I guess it was meant to be alpha. I think/believe the `check_model` should catch the error. If not, we need to fix it.

Reply:
checker does not check for schema error if it is a custom op. I added a check for this type of errors when check a node. I will create another pr to verify consistency between customer ops and local functions in general.

Reply:
https://github.com/onnx/onnx/issues/4957

Reply:
I will create a pr to fix the issue